src/natcap/invest/ui/model.py

```diff
@@ -687,16 +687,7 @@ def set_title_attr(self, name, value):
         """
         LOGGER.info('__setattr__: %s, %s', name, value)
         old_attr = getattr(self, name, 'None')
-        # Python Core and Builtins were updated in Python 3.8.4
-        # that handle __setattr__ differently. In 3.7, the super()
-        # implementation causes a `TypeError: can't apply this __setattr__
-        # to object object` when running `invest run carbon`. In Python 3.8.4+
-        # with the object implementation the error `TypeError: can't apply
-        # this __setattr__ to Carbon object`
-        if (sys.version_info.minor >= 8) and (sys.version_info.micro >= 4):
```

Comment:
For versions 3.9.0 - 3.9.3 and 3.10.0-3.10.3, this would evaluate to `False`, and so the wrong `__setattr__` was used.

Reply:
Whoops!  Thanks for catching this.